pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -129,44 +116,46 @@ stdenv.mkDerivation rec {
     "swresample"
     "swscale"
     "jpeg"
-    "png"
+    # "png"
     # File saving breaks unless we are using vendored llvmPackages_8.libcxx
     #"c++"
-    "ssl" "crypto"
+    "ssl"
+    "crypto"
     "nspr"
     "nss"
     "odbc"
     "tcmalloc" # gperftools
   ];
 
-  installPhase = let
-    steam-run = (steam.override {
-      extraPkgs = p: buildInputs;
-    }).run;
-  in ''
-    prefix=$out/opt/kingsoft/wps-office
-    mkdir -p $out
-    cp -r opt $out
-    cp -r usr/* $out
-    for lib in $unvendoredLibraries; do
-      rm -v "$prefix/office6/lib$lib"*.so{,.*}
-    done
-    for i in wps wpp et wpspdf; do
-      substituteInPlace $out/bin/$i \
-        --replace /opt/kingsoft/wps-office $prefix
-    done
-    for i in $out/share/applications/*;do
-      substituteInPlace $i \
-        --replace /usr/bin $out/bin
-    done
-
-    for i in wps wpp et wpspdf; do
-      mv $out/bin/$i $out/bin/.$i-orig
-      makeWrapper ${steam-run}/bin/steam-run $out/bin/$i \
-        --add-flags $out/bin/.$i-orig \
-        --argv0 $i
-    done
-  '';
+  installPhase =
```

Comment:
It will still be nice to add `runHook preInstall` and `runHook postInstall` before  #175605 is resolved